src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/gadgets/MultiToolMode.java

```diff
@@ -8,19 +8,22 @@
 
 class MultiToolMode {
 
-    private final ItemSetting<String> item;
+    private final String itemId;
     private final ItemSetting<Boolean> enabled;
 
-    MultiToolMode(@Nonnull MultiTool multiTool, int id, @Nonnull String itemId) {
-        this.item = new ItemSetting<>(multiTool, "mode." + id + ".item", itemId);
-        this.enabled = new ItemSetting<>(multiTool, "mode." + id + ".enabled", true);
+    MultiToolMode(@Nonnull MultiTool multiTool, @Nonnull String itemId) {
+        this.itemId = itemId;
+        this.enabled = new ItemSetting<>(multiTool, "mode." + itemId + ".enabled", true);
 
-        multiTool.addItemSetting(item, enabled);
+        multiTool.addItemSetting(enabled);
     }
 
+    @Nonnull
+    String getItemId() { return itemId; }
+    
     @Nullable
     SlimefunItem getItem() {
-        return SlimefunItem.getById(item.getValue());
+        return SlimefunItem.getById(itemId);
     }
 
     boolean isEnabled() {
```

Comment:
Why are we changing this? I think this would be a breaking change config wise.

Reply:
If the reason is to make sure it can't be changed and break existing multi tools you should just make it so that when it looks for a tool if it doesn't find one it resets for the first one

Reply:
For some reason I thought it has nothing to do with the config and I didn't get any errors during testing because of it, so I didn't notice. I'll revert unnecessary changes here.